openapi-generator/src/test/java/io/micronaut/openapi/generator/AbstractMicronautCodegenTest.java

```diff
@@ -37,7 +37,7 @@ protected String generateFiles(MicronautCodeGenerator<?> codegen, String configP
         } catch (IOException e) {
             fail("Unable to create temporary directory for output");
         }
-        output.deleteOnExit();
+//        output.deleteOnExit();
```

Comment:
why?

Reply:
just mistake. I comment it, when I test it locally. Need to see, what were generated. Sometimes I forgot uncomment it